mrp_bom_nested_bom_generator/models/mrp_bom.py

```diff
@@ -0,0 +1,157 @@
+from odoo import _, fields, models
+
+
+class MrpBom(models.Model):
+    _inherit = "mrp.bom"
+
+    bom_type_nested = fields.Boolean(string="Type Nested BOM", default=False)
```

Comment:
Field name can be more straight, string is not clear, "False" is the default value for Booleans
`nested_bom = fields.Boolean(string="Nested BOM")`

Reply:
Resolved